ui/components/app/gas-timing/gas-timing.component.js

```diff
@@ -15,18 +68,10 @@ export default function GasTiming({ text, tooltipText, attitude }) {
       })}
     >
       {text}
-      {tooltipText && <InfoTooltip position="top" contentText={tooltipText} />}
     </Typography>
   );
 }
 
 GasTiming.propTypes = {
-  text: PropTypes.string.isRequired,
-  tooltipText: PropTypes.string,
-  attitude: PropTypes.oneOf(['positive', 'negative', 'warning', '']),
-};
-
-GasTiming.defaultProps = {
-  tooltipText: '',
-  attitude: '',
+  maxPriorityFeePerGas: PropTypes.object,
```

Comment:
Shouldn't this be: `PropTypes.string.isRequired` for now or `PropTypes.number.isRequired` once we only pass numbers in it?